### ISSUE TITLE
Add walltime metrics on sampling

### ISSUE THREAD

**Issue by flaneur2020** [crabml/crabml#72]
Sampling is the process of selecting a token with the highest relative probability from a series of logits

**Comment by guojidan** [comment]
I want implement this 😄

**Comment by Xuanwo** [comment]
> I want implement this 😄

Have fun!

**Comment by flaneur2020** [comment]
sorry for the mix-up, I inadvertently added the sample walltime metric to the optimization in [PR #85](https://github.com/crabml/crabml/pull/85) :pray:
